packages/react-data-grid/package.json

```diff
@@ -19,13 +21,16 @@
     "data-grid",
     "core"
   ],
+  "scripts": {
+    "build": "tsc"
+  },
   "dependencies": {
     "classnames": "^2.2.6",
-    "keymirror": "^0.1.1",
     "prop-types": "^15.7.2",
     "react-is": "^16.8.4",
     "react-is-deprecated": "^0.1.2",
-    "shallowequal": "^1.1.0"
+    "shallowequal": "^1.1.0",
+    "tslib": "^1.9.3"
```

Comment:
Do we need `tslib`?

Reply:
Yes, the `importHelpers` compiler option makes the generated code import helpers from tslib, instead of generating them for each file.
Much like `@babel/plugin-transform-runtime` is generating imports to `@babel/runtime`
https://babeljs.io/docs/en/babel-plugin-transform-runtime
https://www.typescriptlang.org/docs/handbook/compiler-options.html